docs/2.0/reference/pipelines/configurations.md

```diff
@@ -194,6 +178,25 @@ pipelines:
 
 ## Deprecated Configuration Options
 
+#### enable-terragrunt-provider-cache
+
+<HclListItem name="enable-terragrunt-provider-cache" requirement="optional" type="boolean">
+<HclListItemDescription>
+Enables the <span class="external-link"><a href="https://terragrunt.gruntwork.io/docs/features/provider-cache/">Terragrunt Provider Cache</a></span> for plans and applies.
+
+This option is deprecated in favor of setting `TERRAGRUNT_PROVIDER_CACHE` as an [`env`](#env) option.
+</HclListItemDescription>
+<HclListItemDefaultValue defaultValue="false"/>
+<HclListItemExample>
+
+```yaml
+pipelines:
+  enable-terragrunt-provider-cache: true
+```
```

Comment:
It's really awkward that we have _two_ config languages, one "legacy" and one new, but only add values to the legacy one. Why have the new one at all?

Can we add these values to both the new and legacy config going forward, and reference both in all docs?